paper/ms.tex

```diff
@@ -81,46 +80,33 @@
   Evidence for accretion of massive, rocky planetary system in a sun-like pair
 }
 
-\author{
-  Semyeong Oh\altaffilmark{\pu,\lead},
-  Adrian M. Price-Whelan\altaffilmark{\pu},
-  John M. Brewer\altaffilmark{\yale},
-  Keith A. Hawkins\altaffilmark{\columbia},
-  Nathan W. C. Leigh\altaffilmark{\amnh},
-  David W. Hogg\altaffilmark{\ccpp,\mpia,\cca},
-  David N. Spergel\altaffilmark{\pu,\cca},
-  Justin Myles\altaffilmark{\yale} \etal\
-}
+\author{Semyeong Oh} % TODO: need ORCID
+\affil{Department of Astrophysical Sciences, Princeton University, 4 Ivy Lane, Princeton, NJ 08544, USA}
+\affil{To whom correspondence should be addressed: \texttt{semyeong@astro.princeton.edu}}
+
+\author{John M. Brewer} % TODO: need ORCID
+\affil{Department of Astronomy, Yale University, 260 Whitney Ave, New Haven, CT 06511, USA}
+
+\author[0000-0003-0872-7098]{Adrian M. Price-Whelan}
+\affil{Department of Astrophysical Sciences, Princeton University, 4 Ivy Lane, Princeton, NJ 08544, USA}
+
+\author[0000-0003-2866-9403]{David W. Hogg}
+\affil{Center for Computational Astrophysics, Flatiron Institute, 162 Fifth Ave, New York, NY 10010, USA}
+\affil{Center for Cosmology and Particle Physics, Department of Physics, New York University, 726 Broadway, room 1005, New York, NY 10003, USA}
+\affil{Center for Data Science, New York University, 60 Fifth Ave, New York, NY 10011, USA}
+\affil{Max-Planck-Institut f\"ur Astronomie, K\"onigstuhl 17, D-69117 Heidelberg}
+
+\author{David N. Spergel} % TODO: need ORCID
+\affil{Center for Computational Astrophysics, Flatiron Institute, 162 Fifth Ave, New York, NY 10010, USA}
+
+% \author{Keith A. Hawkins}
```

Comment:
I would recommend that we make Keith and Nathan work a little for coauthorship since, right now, everyone else on as coauthor have made clear contributions, whereas theirs are more at the "acknowledgement" level. I would put them in ack's, but circulate to them in the next round - if they provide text or good feedback, bump them up to author status.